tests/gis_tests/gdal_tests/test_srs.py

```diff
@@ -386,3 +394,11 @@ def test_esri(self):
         self.assertIn('DATUM["D_North_American_1983"', srs.wkt)
         srs.from_esri()
         self.assertIn('DATUM["North_American_Datum_1983"', srs.wkt)
+
+    def test_srid(self):
+        """Testing if srid property returns top-level authority code"""
```

Comment:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Testing" since all tests test things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Maybe:
```suggestion
        """The srid property returns top-level authority code."""
```

Reply:
Turns out I did not read all docs thoroughly enough. :innocent: Should we improve the docstrings in the rest of the file as well?

Reply:
> Should we improve the docstrings in the rest of the file as well?

It's not worth changing. We try to avoid bulk code reformatting unless it's a big problem as it creates extra noise in the history.

Reply:
Ok, makes sense. :+1: